docs/en/deploy/Requirements.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: global
+title: Alluxio Software Requirements
+nickname: Software Requirements
+group: Deploying Alluxio
+priority: 9
+---
+
+{:toc}
+
+## General Requirements
+
+There are base requirements for cluster nodes:
+
+* Cluster nodes should be running one of the following supported operating systems:
+  * MacOS 10.10 or later
+  * CentOS - 6.8 or 7
+  * RHEL - 7.x
+  * Ubuntu - 16.04
+* Alluxio is a Java application which requires version 8 of the JRE:
+  * Java JRE 8
+* Alluxio works on IPv4 networks only.
+* Allow the following ports and protocols:
+  * Inbound TCP 22 - ssh as a user to install Alluxio components across specified nodes.
+
+## Master Requirements
+
+There are Alluxio-specific requirements for cluster nodes running the master process.
+
+Note that these are bare minimums to run the software. Running Alluxio at scale and under high load
+will increase these requirements.
+
+* Minimum 4 GB disk space
+* Minimum 4 GB memory
+* Minimum 4 CPU cores
+* Allow the following ports and protocols:
+  * Inbound TCP 19200 - Used as the Alluxio master's for internal leader election port
+  * Inbound TCP 19998 - Used as the Alluxio master's as the RPC port
+  * Inbound TCP 19999 - Used by the Alluxio master's web UI port. Accessible at `http://<master-hostname>:19999`
+  * Inbound TCP 20003 - Used by Alluxio for embedded journal
+
+## Worker Requirements
+
+There are Alluxio-specific requirements for cluster nodes running the worker process:
+
+* minimum 1 GB disk space
+* minimum 1 GB memory
+* minimum 2 CPU cores
+* Allow the following ports and protocols:
+  * Inbound TCP 29998 - Used as the Alluxio worker's RPC port
+  * Inbound TCP 29999 - Used as the Alluxio worker's data transfer port
+  * Inbound TCP 30000 - Used as the Alluxio worker's web UI port. Accessible at `http://<worker-hostname>:30000` in your browser
+  * Inbound TCP 30001 - Used as the Alluxio job worker's RPC port
```

Comment:
@bf8086 @ggezer are we using both RPC and data ports for job worker?

Reply:
ping @bf8086 @ggezer

Reply:
I think job worker gRPC server only uses one port as of 2.0.

Reply:
There's still usage of old property keys for the job workers that call the data+rpc ports. I don't think they are actually used, but we should clean that up in another PR

Reply:
https://github.com/Alluxio/alluxio/blob/master/job/server/src/main/java/alluxio/worker/AlluxioJobWorkerProcess.java#L240-L248